### PR TITLE
FET-1534

### PR DESCRIPTION
**Pull request by yeewiz**
- filtered out the primary name from the list of name addresses
- added unit test to test to make sure the primary name is not within the list

### CODE REVIEW

**Review by storywithoutend — CHANGES_REQUESTED** [review]
A small bug. When searching for the currentPrimaryname, the list area will be blank because the primary name is being filtered out, but the condition for showing the list of names has a length greater than 1.

File: src/transaction-flow/input/SelectPrimaryName/SelectPrimaryName.test.tsx [hunk 1]
```diff
@@ -208,6 +211,25 @@ describe('SelectPrimaryName', () => {
     })
   })
 
+  it('should not show primary name in list', async () => {
+    mockUsePrimaryName.mockReturnValue({
+      data: {
+        name: 'test2.eth',
+        beautifiedName: 'test1.eth',
+      },
```
[the remark this answers is not in the document]
Comment (yeewiz) [reply src/transaction-flow/input/SelectPrimaryName/SelectPrimaryName.test.tsx:219]: doh! changed it to be the same

**Comment by yeewiz** [comment]
> A small bug. When searching for the currentPrimaryname, the list area will be blank because the primary name is being filtered out, but the condition for showing the list of names has a length greater than 1.

should be resolved now @storywithoutend

**Review by storywithoutend — APPROVED** [review]

**Comment by yeewiz** [comment]
didn't want reverts to effect the date branch changes so created new PR: https://github.com/ensdomains/ens-app-v3/pull/764

**Pull request closed without merging**